ui/app/css/itcss/settings/variables.scss

```diff
@@ -58,6 +58,8 @@ $linen: #fdf4f4;
 $oslo-gray: #8C8E94;
 $polar: #fafcfe;
 $blizzard-blue: #bfdef3;
+$spindle: #c6dded;
```

Comment:
We have a `$spindle` already at line 46. Is that `$spindle` too different from the color used in the mock?